### PR TITLE
Add DidiGlobal

### PR DESCRIPTION
**Pull request by RedRubble**
Didi is a transportation app (Chinese Uber), 7,751 top rank. See attachment to see 2FA request. This is not at every login though (similar to AirBnB), and Didi decides when to ask, only seems to ask for SMS in my testing.

DiDi 2FA

### CODE REVIEW

File: img/d/didiglobal.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" style="enable-background:new 0 0 105 85.1" viewBox="0 0 105 85.1"><path d="M388.3 468.2c68-2.3 121.7-56.3 122.5-123.2v-57.8h74.8v63.9c-1.5 103.9-91.8 185.6-197.3 187.8-107.3-.8-198.4-83.1-199.6-187.8V236.6c0-11.7 7.6-19.3 18.9-19.3h303.1v69.9H273.1c-6 0-9.8 3.8-9.8 9.8v46.9c.7 67.6 55.5 123.6 125 124.3z" style="fill-rule:evenodd;clip-rule:evenodd;fill:#fc9153" transform="matrix(.26458 0 0 .26458 -49.927 -57.494)"/></svg>
```
Comment (hkamran80) [inline img/d/didiglobal.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 105 85.1"><path fill="#fc9153" d="M52.8 66.4c18-.6 32.2-15 32.4-32.6V18.5H105v16.9c-.4 27.5-24.3 49.1-52.2 49.7C24.4 84.9.3 63 0 35.4V5.1C0 2.1 2 0 5 0h80.2v18.5H22.3c-1.6 0-2.6 1-2.6 2.6v12.4a33.3 33.3 0 0 0 33.1 32.9z" /></svg>
```

File: entries/d/didiglobal.com.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "Didi": {
+    "domain": "didiglobal.com",
+    "tfa": [
+      "sms"
+    ],
+    "notes": "Didi decides when 2FA is required; user cannot enable 2FA for every login.",
+    "categories": [
+      "transport"
+    ],
+    "regions": [
```
Comment (hkamran80) [inline entries/d/didiglobal.com.json:16]: Do you know if the [Brazilian site](https://99app.com/) and the [Japanese site](https://didimobility.co.jp/) use the same authentication backend?

File: entries/d/didiglobal.com.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+  "Didi": {
```
Comment (hkamran80) [inline entries/d/didiglobal.com.json]:
```suggestion
  "DiDi": {
```

File: entries/d/didiglobal.com.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      "tw",
+      "cl",
+      "cr",
+      "pa",
+      "ru",
+      "pe",
+      "ec",
+      "do",
+      "eg",
+      "ar"
```
Comment (hkamran80) [inline entries/d/didiglobal.com.json]:
```suggestion
      "ar",
      "au",
      "br",
  	  "cl",
  	  "cn",
  	  "co",
  	  "cr",
  	  "do",
  	  "ec",
  	  "eg",
  	  "hk",
  	  "jp",
  	  "mx",
  	  "nz",
  	  "pa",
  	  "pe",
  	  "tw"
```

Here's an alphabetized list of regions

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/d/didiglobal.com.json (hunk 2 above)
Comment (RedRubble) [reply entries/d/didiglobal.com.json:16]: Im not sure, finding information on other countries seems to be very difficult, and their global website is very limited

Comment (hkamran80) [reply entries/d/didiglobal.com.json:16]: Gotcha. I'll do some more research.

Comment (RedRubble) [reply entries/d/didiglobal.com.json:16]: If its known as 99 in Brazil, would I need to create a separate json? and icon?

Comment (hkamran80) [reply entries/d/didiglobal.com.json:16]: If it uses the same authentication backend, that is, you can sign in with the same account, then no. You can add the domain as an [additional domain](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#other-properties).

If it uses a separate account system, then you should create a new entry for that.

I can't tell whether it uses a separate system, so let's add both using `additional-domains` for now.

File: entries/d/didiglobal.com.json [hunk 5]
```diff
@@ -0,0 +1,32 @@
+{
+  "DiDi": {
+    "domain": "didiglobal.com",
+    "url": "https://web.didiglobal.com/"
```
Comment (hkamran80) [inline entries/d/didiglobal.com.json]:
```suggestion
    "url": "https://web.didiglobal.com",
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/d/didiglobal.com.json [hunk 6]
```diff
@@ -0,0 +1,35 @@
+{
+  "DiDi": {
+    "domain": "didiglobal.com",
+    "url": "https://web.didiglobal.com",
+    "additional-domains": [
+      "99app.com"
```
Comment (hkamran80) [inline entries/d/didiglobal.com.json]:
```suggestion
      "99app.com",
      "didimobility.co.jp"
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Review by hkamran80 — APPROVED** [review]

**Comment by hkamran80** [comment]
Thank you! Your contribution will be visible in around 10 minutes, although it may take longer due to [server and browser caching](https://github.com/2factorauth/twofactorauth/wiki/FAQ-Caching).

**Pull request merged**